e2e/features/tour/tour-joyride-test.js

```diff
@@ -19,17 +19,21 @@ module.exports = {
     c.url(`${c.globals.url}?tr=hurricane_dorian_september_joyride&mockTour=true`);
   },
   'Verify that all tour loads properly, Joyride beacon shows after progressing to Step 2': (c) => {
+    c.setWindowSize(1000, 800);
+    c.pause(1500);
     c.useCss().waitForElementVisible('.tour-in-progress .step-total', TIME_LIMIT);
     c.click(tourNext);
-    c.waitForElementVisible(firstBeaconSelector, TIME_LIMIT);
+    c.click('#wv-map');
+    c.pause(500);
+    c.useCss().waitForElementVisible(firstBeaconSelector, TIME_LIMIT);
   },
   'Clicking beacon shows the floater tooltip': (c) => {
     if (c.options.desiredCapabilities.browserName !== 'firefox') { // intermittently fails on Chrome
       return;
     }
-    c.useCss().waitForElementVisible(firstBeaconSelector, TIME_LIMIT);
-    c.pause(500);
-    c.click(firstBeaconSelector);
+    // c.useCss().waitForElementVisible(firstBeaconSelector, TIME_LIMIT);
+    c.pause(5000);
```

Comment:
Another one

Reply:
Removed pause, this was left over from debugging.